proof/invariant-abstract/BCorres_AI.thy

```diff
@@ -14,6 +14,14 @@ abbreviation "bcorres \<equiv> bcorres_underlying truncate_state"
 
 abbreviation "s_bcorres \<equiv> s_bcorres_underlying truncate_state"
 
+lemma get_object_bcorres[wp]:
```

Comment:
I guess we'd like this for when this commit goes into `master`, but while I remember to ask, do we know whether we'd like `bcorres` for MCS?

Reply:
I'm pretty sure it is necessary, unless we have fully made everything deterministic, including the cdt etc.